include/exadg/acoustic_conservation_equations/spatial_discretization/operators/weak_boundary_conditions.h

```diff
@@ -134,13 +153,19 @@ template<int dim, typename Number>
 class BoundaryFaceIntegratorU : public BoundaryFaceIntegratorBase<BoundaryDescriptor<dim>, Number>
 {
   using FaceIntegratorU = FaceIntegrator<dim, dim, Number>;
+  using FaceIntegratorP = FaceIntegrator<dim, 1, Number>;
 
 public:
-  BoundaryFaceIntegratorU(FaceIntegratorU const &         integrator_m_in,
+  BoundaryFaceIntegratorU(FaceIntegratorU const &         integrator_velocity_m_in,
+                          FaceIntegratorP const &         integrator_pressure_m_in,
+                          double const                    speed_of_sound,
```

Comment:
From a pure software design perspective, the dependency on `speed_of_sound` could be seen detrimental. One technical solution to this "problem" could be to integrate the factor `c` into the `dealii::Function` that we already use to prescribe `Y`.

Reply:
What do you think?

Reply:
From a pure software perspective, this is true. I would still like to keep it this way because in literature values for admittance or impedance ($1/Y$) are provided, and it is not intuitive to prescribe $Y/c$ from a physical acoustic perspective.

Reply:
I'm fine with it for now. I take this as an indication that generalization `BoundaryFaceIntegrator`s across modules (acoustics, Poisson, compressible) is not reasonable (for now) for reasons that we also encounter here.